docs/en/observability/ingest-logs.asciidoc

```diff
@@ -0,0 +1,188 @@
+----
+--
+
+To see the full list of variables for an integration, see the documentation under
+{filebeat-ref}/filebeat-modules.html[Modules].
+
+[TIP]
+=====
+To test your configuration file, change to the directory where the Filebeat binary
+is installed, and run Filebeat in the foreground with the following options specified: 
+./filebeat test config -e. Make sure your config files are in the path expected by 
+Filebeat (see {filebeat-ref}/directory-layout.html[Directory layout]),
+or use the -c flag to specify the path to the config file.
+=====
+
+For more information about configuring Filebeat, also see:
+
+* {filebeat-ref}/configuring-howto-filebeat.html[Configure Filebeat]
+* {beats-ref}/config-file-format.html[Config file format]
+* {filebeat-ref}/filebeat-reference-yml.html[`filebeat.reference.yml`]: This reference 
+configuration file shows all non-deprecated options. You’ll find it in the same location as filebeat.yml.
+
+[[set-filebeat-assets]]
+== Step 4: Set up assets
+{beatname_uc} comes with predefined assets for parsing, indexing, and
+visualizing your data. To load these assets:
+
+. Make sure the user specified in +{beatname_lc}.yml+ is
+{filebeat-ref}/privileges-to-setup-beats.html[authorized to set up {beatname_uc}].
+
+. From the installation directory, run:
++
+--
+include::{beats-repo-dir}/tab-widgets/setup-widget.asciidoc[]
+--
++
+`-e` is optional and sends output to standard error instead of the configured log output.
+
+This step loads the recommended {ref}/indices-templates.html[index template] for writing to {es}
+and deploys the sample dashboards for visualizing the data in {kib}.
+
+This step does not load the ingest pipelines used to parse log lines. By
+default, ingest pipelines are set up automatically the first time you run the
+module and connect to {es}.
+
+[TIP]
+=====
+A connection to {es} (or {ess}) is required to set up the initial
+environment. If you're using a different output, such as {ls}, see:
+
+* {filebeat-ref}/filebeat-template.html#load-template-manually[Load the index template manually]
+* {filebeat-ref}/load-kibana-dashboards.html[Load Kibana dashboards]
+* {filebeat-ref}/load-ingest-pipelines.html[Load ingest pipelines]
+=====
+
+[[start-filebeat]]
+== Step 5: Start {beatname_uc}
+
+Before starting {beatname_uc}, modify the user credentials in
++{beatname_lc}.yml+ and specify a user who is
+{filebeat-ref}/privileges-to-publish-events.html[authorized to publish events].
+
+To start {beatname_uc}, run:
+
+// tag::start-step[]
+:requires-sudo:
+include::{beats-repo-dir}/tab-widgets/start-widget.asciidoc[]
+:requires-sudo!:
+// end::start-step[]
+
+{beatname_uc} should begin streaming events to {es}.
+
+[[view-logs-kibana]]
+== Step 6: View your data in {kib}
+
+To view the <<observability-ui,Observability Overview>> page:
+
+. Launch {kib}:
++
+--
+include::{beats-repo-dir}/tab-widgets/open-kibana-widget.asciidoc[]
+--
+
+. In the side navigation, select *Observability*, and click *Overview*. 
```

Comment:
Is this 7.8? If so, this is not correct. I haven't looked at 7.9 yet, so disregard if this is correct for 7.9.

Reply:
Yes this is just for the 7.9 release, not 7.8. It's the new Observability Overview page.